packages/integration-karma/test/polyfills/aria-properties/index.spec.js

```diff
@@ -1,7 +1,7 @@
 function testAriaProperty(property, attribute) {
     describe(property, () => {
         it(`should assign property ${property} to Element prototype`, () => {
-            expect(Element.prototype.hasOwnProperty(property)).toBe(true);
+            expect(Object.prototype.hasOwnProperty.call(Element.prototype, property)).toBe(true);
```

Comment:
why was this a problem?

Reply:
I didn't want to hook everything up to make @lwc/shared available in the karma env :)